dask/bytes/tests/test_http.py

```diff
@@ -195,6 +195,7 @@ def test_parquet(engine):
     reruns=10, reruns_delay=5, reason="https://github.com/dask/dask/issues/3696"
 )
 @pytest.mark.network
+@pytest.mark.skip(reason="Flaky test.")
```

Comment:
All my `requests`-based HTTP requests to `https://en.wikipedia.org` are blocked with a 403 and the message `Please set a user-agent and respect our robot policy https://w.wiki/4wJS. See also T400119.`.

Maybe worth resurrecting the test, but not right now.